packages/core/src/tools/create-file-stasher.js

```diff
@@ -151,7 +151,6 @@ const createFileStasher = input => {
             };
 
             if (isStreamed) {
-              maybeResponse.throwForStatus();
```

Comment:
I'm not 💯 on this one, but I _think_ any `maybeResponse` will also have gone through `afterResponse`.